openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -152,9 +152,13 @@ <h4 class="subsection-title">
                                 >
                     % for vertical in subsection.get('children', []):
                                     <li class="vertical outline-item focusable">
+                                    % if preview:
+                                        <span class="outline-item">
```

Comment:
Not setting the `href` may make more sense than changing the element type (though not sure what the a11y requirements are). If you want to go with the `span`, the styling needs to be updated. Currently, the unit items are all showing in a single line.

Reply:
That's odd, I've updated the scss and this was working here. I'll review that approach.

Reply:
Hmm. Okay, it could just be that my assets were not updated correctly. I'll check it again in the next pass.